docs/addons/karpenter.md

```diff
@@ -23,7 +23,23 @@ import * as blueprints from '@aws-quickstart/eks-blueprints';
 
 const app = new cdk.App();
 
-const addOn = new blueprints.addons.KarpenterAddOn();
+const karpenterAddonProps = {
+  ProvisionerSpecs: {
```

Comment:
This needs an update as well to match the code.

Reply:
addressed